src/couch/src/couch_debug.erl

```diff
@@ -326,6 +329,41 @@ help(print_table) ->
           - TableSpec: List of either {Value} or {Width, Align, Value}
             where Align is either left/center/right.
 
+        ---
+    ", []);
+help(print_report) ->
+    io:format("
+        print_report(Report)
+        --------------------------------
+
+        Print a report in table form.
+          - Report: List of {InfoKey, InfoVal} where each InfoKey is unique
+          (unlike print_table/2).
+
+        The output will look similar to:
+
+            |info           |                                                                                               value
+            |  btree_size   |                                                                                                  51
+            |  def          |                                                                     function(doc){emit(doc.id, 1);}
```

Comment:
I am curious what we should do for the case when the JavaScript function is longer than table width. Should we truncate the values?

Reply:
I agree with truncating it, but the question is in which way? How do we distinguish what is important enough to return to the user and what is not?